### PR TITLE
Add notes about aks vmss to azure readme.

### PR DESCRIPTION
**Pull request by paulgmiller**
VMSS names and resource group aren't clear from the docs so try ad improve

### CODE REVIEW

File: cluster-autoscaler/cloudprovider/azure/README.md [hunk 1]
```diff
@@ -42,25 +42,27 @@ Fill the values of cluster-autoscaler-azure secret in [cluster-autoscaler-vmss.y
...
 
-or multiple node groups:
+or multiple vm scale sets:
 
 ```yaml
         - --nodes=1:10:vmss1
         - --nodes=1:10:vmss2
 ```
 
+Some aks notes: ResourceGroup will be the rg created by aks (usually MC_<cluster>_<region>) ratehr than the group the aks resource is in. The vm scale set names will be aks-<nodepool>-<hash>-vmss. Where the function that computes that hash is under flux
```
Comment (feiskyer) [inline cluster-autoscaler/cloudprovider/azure/README.md]: Thanks for adding this.

**Comment by feiskyer** [comment]
@paulgmiller Thanks for updating the docs. Could you sign the CLA?

**Comment by paulgmiller** [comment]
I'm actually a microsoft employee so I think I need to get legal to white list me?

**Comment by paulgmiller** [comment]
Okay CLA signed.

**Pull request merged**